shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -180,3 +180,39 @@ subquery
 lockClause
     : FOR UPDATE 
     ;
+
+merge
+    : MERGE hint? intoClause usingClause mergeUpdateClause? mergeInsertClause? errorLoggingClause?
+    ;
+
+hint
+    : (BLOCK_COMMENT | INLINE_COMMENT)
+    ;
+
+intoClause
+    : INTO (tableName | viewName) alias?
+    ;
+
+usingClause
+    : USING ((tableName | viewName) | subquery) alias? ON LP_ expr RP_
+    ;
+
+mergeUpdateClause
+    : WHEN MATCHED THEN UPDATE SET mergeAssignment (COMMA_ mergeAssignment)* whereClause? (DELETE whereClause)?
+    ;
+
+mergeAssignment
+    : columnName EQ_ mergeAssignmentValue
+    ;
+
+mergeAssignmentValue
+    : (expr | DEFAULT)
+    ;
+
+mergeInsertClause
+    : WHEN NOT MATCHED THEN INSERT (LP_ columnName (COMMA_ columnName)* RP_)? VALUES LP_ mergeAssignmentValue (COMMA_ mergeAssignmentValue)* RP_ whereClause?
```

Comment:
'mergeAssignmentValue' is not suitable.

Reply:
What do I need to change here?

Reply:
![image](https://user-images.githubusercontent.com/22066046/108449658-ccb8e280-729e-11eb-8389-3bbcf67524a8.png)
‘mergeAssignmentValue’ could be used in merge_update_clause, not merge_insert_clause. You should write a new one, like mergeColumnValue in merge_insert_clause.

Reply:
In the `mergeInsertClause`, my `ANTLR` definition for the first set is `WHEN NOT MATCHED THEN INSERT (LP_ columnName (COMMA_ columnName)* RP_)?` 

I used `mergeAssignmentValue` in the second set because just not to repeat `(expr | DEFAULT)` 

So, defining a new rules like `mergeColumnValue : VALUES LP_  (expr | DEFAULT) (COMMA_  (expr | DEFAULT))* RP_`,  and something like `mergeInsertColumn : (LP_ columnName (COMMA_ columnName)* RP_)?` would that be okay?

Reply:
Yes